test/browser/fixtures/server-url/vitest.config.ts

```diff
@@ -9,6 +9,16 @@ import basicSsl from '@vitejs/plugin-basic-ssl'
 const provider = process.env.PROVIDER || 'webdriverio';
 const browser = process.env.BROWSER || (provider === 'playwright' ? 'chromium' : 'chrome');
 
+// ignore https errors due to self-signed certificate from plugin-basic-ssl
+// https://playwright.dev/docs/api/class-browser#browser-new-context-option-ignore-https-errors
+// https://webdriver.io/docs/configuration/#strictssl and acceptInsecureCerts in https://webdriver.io/docs/api/browser/#properties
+const providerOptions = (function () {
+  switch (process.env.PROVIDER) {
```

Comment:
Could you change it to `provider` so that it'll pick up `webdriver` when `process.env.PROVIDER` is undefined?

```suggestion
  switch (provider) {
```

Reply:
Oh, for sure, dunno what I was thinking